bindings/rust/aws-lc-sys-template/src/lib.rs

```diff
@@ -1,22 +1,24 @@
-#![allow(non_upper_case_globals)]
-#![allow(non_camel_case_types)]
-#![allow(non_snake_case)]
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
 
-// populated by cmake
-${INCLUDES}
+mod bindings;
+pub use bindings::*;
 
+#[allow(non_snake_case)]
 pub fn ERR_GET_LIB(packed_error: u32) -> i32 {
```

Comment:
What makes these three functions special compared to the rest of libcrypto?

Reply:
These "functions" are actually preprocessor macros. An alternative way would be to code the operations of the Macro in Rust (as is done [here](https://github.com/cloudflare/boring/blob/master/boring-sys/src/lib.rs#L47)), but if the macro was ever changed we might forget to update that logic in Rust. The way it's done in this PR (utilizing Boring's `rust_wrapper.c` above) helps ensure that the logic used in Rust is consistent with the C macro.

Reply:
Honestly, I don't really see those macros ever changing their logic. There might be a slight performance improvement by implementing it in Rust directly? I don't have a strong opinion on this either way.